docs/aerodromes/classc/Sydney.md

```diff
@@ -283,59 +283,115 @@ Climb Gradient Requirements apply to all Procedural SIDs. It is the pilot's resp
 | Between 2000FT & 3000FT| >5000M         | `EXP INSTR APCH THEN INDEP VISUAL APCH WHEN VISUAL`|
 | Below 2000FT **or**    | <5000M         | `EXP INSTR APCH`                |
 
-When Independent Visual Approaches are run during PROPS, the APCH field must also include:  
-`DO NOT PASS THRU ASSIGNED RWY CL`
+!!! Important
+    Independent Visual Approaches (IVAs) may only be run during Parallel Runway Operations (PROPS). During single runway operations, regular visual approaches (blank ATIS 'APCH' field) should be used instead.
+
+When Independent Visual Approaches are run during PROPS, the APCH field must also include `DO NOT PASS THRU ASSIGNED RWY CL`.
 
 ### Operational Info
 
-When parallel runways are used for departures but inbound traffic levels don't necessitate independent approaches (low traffic levels), the ATIS OPR INFO shall include:  
-`INDEP PARL DEPS IN PROG`  
+> For the following approach types:
+> 
+> - `EXP INSTR APCH`
+> 
+> - `EXP INSTR APCH THEN INDEP VISUAL APCH WHEN VISUAL`
+> 
+> the ATIS OPR INFO shall include `INDEP PARL DEPS IN PROG`.
+> 
+> !!! Note
+    This permits independent departures (two simultaneous departures on parallel runways) but NOT independent approaches (two simultaneous arrivals on parallel runways without perscribed separation minima).
+
+⠀  
+
+> For the following approach type,
+> 
+> - `EXP INDEP VISUAL APCH`
+> 
+> the ATIS OPR INFO shall include `INDEP PARL APPROACHES AND DEPS IN PROG`.
+
+⠀  
+
+> When weather conditions are very poor (such as during thunderstorms), the use of independent approaches and departures are prohibited and the ATIS OPR INFO shall include `PARL RWY OPS IN PROGRESS`.
```

Comment:
Better description for "very poor". Needs to be specific

Reply:
I've spoken to a few people and I don't believe we have the specific threshold used IRL. Might make one up for now and perhaps send an FOI to Airservices and see if we can get the actual figure

Reply:
I think a good enough threshold would be "if thunderstorms are on the ATIS". My best guess would be that rule is in place, in case a 34L/16L departure has to deviate right due weather, or a 34R/16R departure has to deviate left due weather. TS is the best predictor of weather deviations.

OR we could remove that rule all together, in an attempt to simify operations for VATSIM at an already complicated aerodrome, in an environment where weather deviations are very rare

Reply:
I think the realism is great but also value simplicity to improve the odds of a controller adhering to the published procedures. I'm happy to remove this particular clause and just leave it at IAP, IVA and IAP then IVA when visual. That covers 99% of operations and helps to declutter this page a little.

Reply:
And if traffic congestion in the TMA is getting out of hand (due to weather avoidance or otherwise), DEPs can cancel autorelease and control the rate that way

Reply:
Apologies; didn't see the further messages in this thread & just committed changes that specify adverse weather conditions of thunderstorms or heavy rain with a vis of less than 500m. Thoughts on this?

Reply:
I would still prefer to remove this stipulation in order to reduce complexity

Reply:
Sounds good